core/src/main/scala/com/pingcap/tispark/v2/TiDBTable.scala

```diff
@@ -103,6 +107,25 @@ case class TiDBTable(
   def logicalPlanToRDD(dagRequest: TiDAGRequest, output: Seq[Attribute]): List[TiRowRDD] = {
     getLogicalPlanToRDD(dagRequest, output, session, sqlContext, tableRef)
   }
+
+  override def newWriteBuilder(info: LogicalWriteInfo): WriteBuilder = {
+    var scalaMap = info.options().asScala.toMap
+    // insert sql without option
+    if (scalaMap.isEmpty) {
+      throw new TiBatchWriteException(
+        "option is neccessary. maybe you are using insert sql, it's not supported yet")
```

Comment:
The initial letter should be uppercase.
Please check the whole PR.
`maybe you are using insert sql, it's not supported yet`, I can't get it.

Reply:
Insert SQL will go into this method, but it will fail because of the missing of option